ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.java

```diff
@@ -50,17 +53,33 @@ public String getName() {
     return "StatusBarManager";
   }
 
-  @Override
-  public @Nullable Map<String, Object> getConstants() {
+  // TODO(dantman): Detect rare status bar height changing events (such as a Phone+Tablet or docking device that can change
```

Comment:
Please remove this todo. Without a task number it will not pass linter.

Reply:
Shame, StatusBar.js had a similar type of comment in it.